README.md

```diff
@@ -1,177 +1,99 @@
-```
+## Non-goals
 
-#### Use case 2
+ * Generalize the header to signal more than the intended hint.
+ * Allow explicit control of the speculative parsing or preloading scanning mechanism — it’s better this is designed as a hint that a response is supplied with, on which the UA may choose to vary its usage of speculative parsing on.
 
-[etc.]
+## User research
 
-## Detailed design discussion
+An Origin Trial was conducted in Chrome from M125 to M130 that allowed participants to disable the default speculative prefetch behavior of the user agent. More information about the Origin Trial can be found on the [Chrome Platform Status page for the Skip Preload Scanning Feature](https://chromestatus.com/feature/5190976638550016) and the [explainer document](https://docs.google.com/document/d/1wiaTL5TeONTZamycMVMjo76nMcbhHNYznQy7I_zCVRY/edit).
 
-### [Tricky design choice #1]
+Additional comments and usecases from the community were received on the [tracker issue](https://issues.chromium.org/issues/330802493) that influenced the design of this header.
 
-[Talk through the tradeoffs in coming to the specific design point you want to make.]
+Participants observed improved performance on pages that do not have external
+resource fetches by reducing the time spent towards speculative parsing.
 
-```js
-// Illustrated with example code.
-```
+## Specification plan
 
-[This may be an open question,
-in which case you should link to any active discussion threads.]
+The specification for this feature in itself consists of two parts.
 
-### [Tricky design choice 2]
+One part is header parsing, which relies on the [Structured Headers](https://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html) specification to do the work. The other is altering behavior of the [active HTML speculative parser](https://html.spec.whatwg.org/multipage/parsing.html#active-speculative-html-parser), which would be done by modifying the HTML Standard's [Speculative HTML parsing](https://html.spec.whatwg.org/multipage/parsing.html#speculative-html-parsing).
 
-[etc.]
+Please see the associated [spec proposal](https://explainers-by-googlers.github.io/prefer-no-speculative-parsing) for more information.
 
 ## Considered alternatives
 
-[This should include as many alternatives as you can,
-from high level architectural decisions down to alternative naming choices.]
-
-### [Alternative 1]
-
-[Describe an alternative which was considered,
-and why you decided against it.]
-
-### [Alternative 2]
+### A `<meta>` tag version
 
-[etc.]
+While a `meta` header is in some cases easier for web developers to add to their page than a header, it would make the behavior of such a directive unpredictable. It will also add further complexity in determining how such a directive would work given the placement and timing of itself against resources specified in the HTML. This option has been considered and rejected.
```

Comment:
```suggestion
While a `meta` element is in some cases easier for web developers to add to their page than a header, it would make the behavior of such a directive unpredictable. It will also add further complexity in determining how such a directive would work given the placement and timing of itself against resources specified in the HTML. This option has been considered and rejected.
```